### ISSUE TITLE
WebSocket Connection Failed after Update to V2.5.0

### ISSUE THREAD

**Issue by KiddoV** [wailsapp/wails#2667]
### Description

Not sure what happen, but after update Wails to v2.5.0, my Svelte app always yell this error in console:
```console
WebSocket connection to 'ws://wails.localhost/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED
setupWebSocket @ client.ts:78
(anonymous) @ client.ts:68
client.ts:59 [vite] Direct websocket connection fallback. Check out https://vitejs.dev/config/server-options.html#server-hmr to remove the previous connection error.
```

### To Reproduce

N/A

### Expected behaviour

No Error when app first init.

### System Details

```shell
wails doctor
DEB | Using go webview2loader
Wails CLI v2.5.0

 SUCCESS  Done.

# System

OS           | Windows 10 Enterprise LTSC 2021
Version      | 2009 (Build: 19044)
ID           | 21H2
Go Version   | go1.19
Platform     | windows
Architecture | amd64

# Wails

Version | v2.5.0

# Dependencies

Dependency | Package Name | Status    | Version      
WebView2   | N/A          | Installed | 113.0.1774.42
Nodejs     | N/A          | Installed | 16.17.0
npm        | N/A          | Installed | 8.15.0
*upx       | N/A          | Installed | upx 3.96
*nsis      | N/A          | Available | 
* - Optional Dependency

# Diagnosis

Your system is ready for Wails development!
Optional package(s) installation details:
  - nsis : More info at https://wails.io/docs/guides/windows-installer/
```

**Comment by stffabi** [comment]
Thanks for opening this issue. 

Unfortunately `2.5.0` has broken the DevServer on most platforms and the next patch release `2.5.1` should be coming pretty soon.

PR #2664 should fix all those DevServer problems. So if you have any spare time, it would be awesome if you could test that PR in your case.

**Comment by KiddoV** [comment]
I confirmed this works now, at least in my case.
Don't see no error anymore.
Thanks!

**Comment by duolabmeng6** [comment]
I also encountered this problem...

**Comment by KiddoV** [comment]
This should already been fixed on v2.5.1
